tools/tcat_ble_client/cli/base_commands.py

```diff
@@ -103,6 +105,38 @@ async def execute_default(self, args, context):
         return CommandResultTLV(tlv_response)
 
 
+class PingCommand(Command):
+
+    def get_help_string(self) -> str:
+        return 'Echo request to device.'
+
+    async def execute_default(self, args, context):
+        bless: BleStreamSecure = context['ble_sstream']
+        size = 10
+        max_payload = 512
+        if len(args) > 0:
+            size = int(args[0])
+            if size > max_payload:
+                print(f'Payload to large. Maximum supported value is {max_payload}')
+                return
+        to_send = token_bytes(size)
+        data = TLV(TcatTLVType.PING.value, to_send).to_bytes_extended()
```

Comment:
could use the extended TLV type only if the payload size is > 254 -> that may help to test the TCAT device as DUT, so we can check it supports both base and extended TLV types.   Ideally the TLV class would encode automatically to base or extended type depending on need.

Reply:
I should work fine now.